### PR TITLE
[ci] Migrate azp from sonicbld agent pool to sonicbld-1ES.

### PR DESCRIPTION
**Pull request by liushilongbuaa**
#### Why I did it
Azure will block self-hosted agent pools.
Migrate to 1ES agent pool to avoid being blocked.
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

### CODE REVIEW

File: .azure-pipelines/official-build.yml [hunk 1]
```diff
@@ -34,11 +34,14 @@ variables:
...
 - stage: Build
-  pool: sonicbld
+  pool: sonicbld-1ES
   variables:
   - name: CACHE_MODE
     value: wcache
   - template: azure-pipelines-repd-build-variables.yml
+  preSteps:
+  - script: if which init.sh;then init.sh;fi
+    displayName: Init agent
```
Comment (xumia) [inline .azure-pipelines/official-build.yml:44]: How about add a template for it, if we want to add more script, can change it easy.

**Comment by liushilongbuaa** [comment]
AA

**Comment by liushilongbuaa** [comment]
Me too

**Comment by liushilongbuaa** [comment]
Me too

**Pull request closed without merging**